site/src/pages/TemplatesPage/TemplatesPageView.tsx

```diff
@@ -115,10 +115,10 @@ export const TemplatesPageView: FC<React.PropsWithChildren<TemplatesPageViewProp
         <Table>
           <TableHead>
             <TableRow>
-              <TableCell>{Language.nameLabel}</TableCell>
-              <TableCell>{Language.usedByLabel}</TableCell>
-              <TableCell>{Language.lastUpdatedLabel}</TableCell>
-              <TableCell>{Language.createdByLabel}</TableCell>
+              <TableCell width="50%">{Language.nameLabel}</TableCell>
+              <TableCell width="16%">{Language.usedByLabel}</TableCell>
```

Comment:
It looks great, but I wonder if there's a better way than hard-coding a percentage.

Reply:
The MUI docs didn't give me any good ideas (you can set a cell to "small" but the problem was they were too small, even though the default is "medium", so that wouldn't help), so I think this is fine as long as it looks good in different screen sizes!